alcs-api/src/board/board.controller.ts

```diff
@@ -60,9 +60,9 @@ export class BoardController {
       planningReviews = await this.planningReviewService.getCards();
     }
 
-    let amendments: ApplicationAmendment[] = [];
+    let modifications: ApplicationModification[] = [];
     if (boardCode === 'ceo') {
```

Comment:
We need gradually get rid of hardcoded values and switch to enums.

Reply:
Added enum for board codes